test-directory/tests.sh

```diff
@@ -1,3 +1,9 @@
 #!bin/sh
 emacs --batch --eval '(message (pp (+ 2 2)))'
 
+# test for Download last uploaded file 
+echo "let plus = ( + )">expected-demo.ml
+echo -ne '\ntest\n' | learn-ocaml-client --server=http://localhost --html --id=demo expected-demo.ml 
+emacs -l learn-ocaml.el -f learn-ocaml-download-server-file :id "demo" &
+sleep 10
+diff demo.ml expected-demo.ml
```

Comment:
Question: shouldn't this fail? because demo.ml and expected-demo.ml don't have the same number of lines… demo.ml seems to have two empty lines after `let plus = ( + )`

Reply:
I will see it. It's normal that the test pass because the tests are not executed. In travis it says 0.07 s when it has a sleep 10

Reply:
ok… let me informed if you figure out why this doesn't work

Reply:
https://github.com/pfitaxel/learn-ocaml.el/pull/3#discussion_r291122400